lib/dataset/utils/parsers.js

```diff
@@ -106,7 +106,7 @@ function parseDoubleGroupedMetric(){
     var dataset = new Dataset()
       .type('double-grouped-metric');
     each(res.result, function(record, i){
-      dataset.set([ 'Result', record[options[0][0]] + ' ' + record[options[0][1]] ], record.result);
+      dataset.set([ 'Result', String(record[options[0][0]] + ' ' + record[options[0][1]]) ], record.result);
```

Comment:
Yep, good catch.. There was a bug previously where labels set as numbers or `null`s throws everything off. Might move this into Dataset as a last measure to ensure string coercion.